src/interface_py/setup.py

```diff
@@ -108,9 +95,13 @@ class BinaryDistribution(Distribution):
     def is_pure(self):
         return False
 
+# Read version
+about_info={}
+with open('__about__.py') as f: exec(f.read(), about_info)
+
 setup(
     name='h2o4gpu',
-    version='0.0.4',
+    version=about_info['__version__'],
     author='H2O.ai, Inc.',
     author_email='h2ostream@googlegroups.com',
```

Comment:
Not sure if this link is still useful. We can check with @ledell to see whats a proper link for support. Not important now,but if we go to pypi then it would be more important.